app/src/main/java/com/vci/vectorcamapp/core/domain/model/Specimen.kt

```diff
@@ -2,5 +2,6 @@ package com.vci.vectorcamapp.core.domain.model
 
 data class Specimen(
     val id: String,
-    val remoteId: Int?
+    val remoteId: Int?,
+    val wasSelectedForFurtherProcessing: Boolean = false
```

Comment:
For domain models, we should not use default values. This will help avoid the issue where we accidentally forget to define a property. In this case android studio will highlight the issue with an error. You should remove the default value and update the instances of Specimen throughout the codebase to help account for the creation of new specimens. For now, you can set all instances to false. When we implement the randomization algorithm, we can change to true based on the probability.